src/serializers/PnmlSerializer.js

```diff
@@ -2,6 +2,7 @@ export default class PnmlSerializer {
 
     constructor () {
         this.xmlSerializer = new XMLSerializer();
+        this.idMap = {};
```

Comment:
why not `this.ids`?

Reply:
Because it's used as a map. :laughing:

Reply:
But its still an object. There is no map in JavaScript :D

Reply:
Ah, sry, my mistake, there is a Map in JavaScript, but this is just an Object.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
Everything in JS is just an object. :smile: And because of this I see no harm in using the variable name to hint at the objects semantics ...

Reply:
idc, I'll merge and publish it.